README.md

```diff
@@ -50,6 +50,14 @@ Available options:
 When setting query parameters as part of your URL, ensure they are encoded correctly. In JS,
 this would be `encodeURIComponent(myURLWithParams)`.
 
+### Explicit rendering event
+In some cases, like if you have a video playing on your page, it can be difficult to determine
+when your page has loaded. You can fire an event to indicate that the page has completed
+loading and rendering.
```

Comment:
Rewrote this whole thing to explain how the loading function works and how you can then choose to fire an event if you like.